main.star

```diff
@@ -69,6 +69,11 @@ def run(plan, args={}):
     )
     prometheus_additional_metrics_jobs = []
 
+    raw_jwt_secret = read_file(static_files.JWT_PATH_FILEPATH)
```

Comment:
Can you change the README to show these new configurations?

Reply:
nobody should have a different experience to how to it was done before. It should be hidden from the user. We didn't specify how jwt was generated before. Why do you think we should do it now?

Reply:
I chose this random line but i meant the entire thing; the support of devnets and the ability to pass in `network_params.network`

Reply:
ah yeah thats in the other PR already.